kafka-test/CMakeLists.txt

```diff
@@ -32,6 +32,9 @@
 #      Cyber and Information Security Research (CISR) Group Oak Ridge National
 #      Laboratory
 #      865-804-5161 (mobile)
+
+# NOTE: THIS FILE IS DEPRECATED AS OF 5/17/2023 AND IS NOT RECOMMENDED FOR USE.
```

Comment:
Why is this deprecated?

Reply:
It is added here. https://github.com/usdot-jpo-ode/asn1_codec/blob/30e2b4d611b04161b42b535b9329f68aad443de2/CMakeLists.txt#L58
If we run the cmake command with higher cmake version 3 at the root folder, this cmake 2.6 will report cmake version is low. 
![image](https://github.com/usdot-jpo-ode/asn1_codec/assets/62157949/09fac873-5c41-4c71-a484-7ff1aa211a9f)

Reply:
I stand corrected, the deprecation note has been removed.